src/components/LoanCollections/KivaClassicSingleCategoryCarousel.vue

```diff
@@ -30,9 +22,7 @@
 </template>
 
 <script>
-import IconClimateChallenge from '@/assets/icons/inline/eco-challenge/globe-leaf.svg';
 import KivaClassicLoanCarousel from '@/components/LoanCollections/KivaClassicLoanCarousel';
-import KvPill from '@/components/Kv/KvPill';
```

Comment:
It looks like we don't use this component elsewhere, safe to delete the component entirely?

Reply:
I'd like to keep these around for now as I think we'll prob revisit when we fill out our new component lib.

Reply:
Yeah these are actually quite nice looking.